spec/features/medicaid_application_with_multiple_members_spec.rb

```diff
@@ -134,16 +134,9 @@
 
     on_page "Health Coverage Needs" do
       expect(page).to have_content(
-        "Tell us what insurance plan each person is enrolled in.",
+        "What type of insurance plan is Jessie Tester currently enrolled in",
```

Comment:
should we update this spec to include multiple `insured` members so that we can confirm the `next` logic is working as-expected?